guides/source/deploy_performance_tuning.md

```diff
@@ -0,0 +1,146 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Tuning Performance for Deployment
+=================================
+
+This guide covers performance and concurrency configuration for deploying your production Ruby on Rails application.
+
+After reading this guide, you will know:
+
+* Whether to use Puma, the default application server
+* How to configure important performance settings for Puma
+
+More information about how to configure your application can be found in the [Configuration Guide](configuring.html).
+
+--------------------------------------------------------------------------------
+
+Choosing an Application Server
+------------------------------
+
+An application server uses a particular concurrency method. For example Unicorn uses processes, Puma is hybrid process- and thread-based concurrency, Thin uses EventMachine and Falcon uses Ruby Fibers.
```

Comment:
```suggestion
An application server uses a particular concurrency method. For example Unicorn uses processes, Puma is hybrid process- and thread-based concurrency and Falcon uses Ruby Fibers.
```

Not sure if it's still worth mentioning Thin today. However it might make sense to mention Passenger: https://rails-hosting.com/2022/#which-rails-servers-are-you-using-in-production

Reply:
I mostly mention Thin because it's mentioned in other Guides, so I know it's okay to mention, and because it has an unusual concurrency method (like Falcon for Fibers). But yeah, not common in production. I've left Thin, but included Passenger.